### PR TITLE
UUID keys

### PR DESCRIPTION
**Pull request by diegolovison**

### CODE REVIEW

**Comment by alanfx** [comment]
Looks good, @diegolovison!

File: extensions/cache/src/main/java/org/radargun/stages/cache/test/UUIDKeysSelector.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import org.radargun.stages.test.TestStage;
+
+public class UUIDKeysSelector implements KeySelector {
+
+   public UUIDKeysSelector() {
+   }
+
+   @Override
+   public long next() {
+      return UUID.randomUUID().getMostSignificantBits() & Long.MAX_VALUE;
```
Comment (rvansa) [inline extensions/cache/src/main/java/org/radargun/stages/cache/test/UUIDKeysSelector.java:17]: Oh my, folks - don't use `UUID.randomUUID()` in performance tests! This uses secure source of randomness and consumes entropy. Eventually you could run out and get blocked in this call.

Check out https://www.baeldung.com/java-security-egd

Comment (diegolovison) [reply extensions/cache/src/main/java/org/radargun/stages/cache/test/UUIDKeysSelector.java:17]: How about
```java
long next = ThreadLocalRandom.current().nextLong() & Long.MAX_VALUE;
```

Comment (rvansa) [reply extensions/cache/src/main/java/org/radargun/stages/cache/test/UUIDKeysSelector.java:17]: Yes, please do that.

**Pull request merged**